src/edge_index.cpp

```diff
@@ -15,6 +15,11 @@
 #include <set>
 #include <unordered_map>
 #include <vector>
+#include <array>
+
+#include <nlohmann/json.hpp>
+#include <fstream>
+#include <unistd.h>
```

Comment:
`unistd.h` inclusion should be guarded for Windows :)

Reply:
used to generate a filename for some debug output.

Reply:
Let me stress once more that while this does reduce the time to read synapses by a factor 8 - 10. This isn't mergeable.

Reply:
No more `<unistd.h>` :)